content/release-10-10-0/applications-10-10-0-bundle/10-10-0-8.md

```diff
@@ -185,7 +185,7 @@ MTM-38172</td>
 <tr>
 <td>
 Export</td>
-<td > Measurements added from data points are now escaped with bracket notation in order to support special characters (e.g. whitespace) in fragment and series names.</td>
+<td > Measurements added from data points are now escaped with bracket notation in order to support special characters (such as whitespace) in fragment and series names.</td>
```

Comment:
I would recommend to use "(for example, whitespace)" (also below) as that is also used in the rest of documentation. "Such as" is used without parentheses.

Reply:
fixed